frontend/src/libs/components/modal/styles.module.scss

```diff
@@ -4,6 +4,7 @@
   align-items: center;
   background: none;
   border: none;
+  outline: none;
```

Comment:
```suggestion
```

Reply:
if i not add this,when you click esc you wiil be have border from you browser
![outline](https://user-images.githubusercontent.com/49407959/230576507-d2e4b754-f508-4a2c-b56b-b3c5273cbdb8.png)

Reply:
and do i need fix problem that write me linter and add reference.current  inside useEffect like const (const modalElement = reference.current) and after in return will remove  event of him

Reply:
https://www.ibm.com/docs/en/raa/6.1?topic=metrics-cyclomatic-complexity

Reply:
you should not delete this